src/tests/unit/tests/injected/analyzer-controller.test.ts

```diff
@@ -4,10 +4,6 @@ import { Requirement } from 'assessments/types/requirement';
 import { FeatureFlagStore } from 'background/stores/global/feature-flag-store';
```

Comment:
Oh, these changes were a result of using the message configuration property on the switcher nav which is no longer used. Since I was updating the switcher nav, I figured I'd remove that property (and its mentions).